.github/README.md

```diff
@@ -42,6 +42,10 @@ In order to use that new clang compiler and archiver specify it when runing `mak
 
     AR=/opt/homebrew/opt/llvm/bin/llvm-ar CC=/opt/homebrew/opt/llvm/bin/clang make wasm
 
+For Intel Macs where the homebrew commands are not installed in /opt/homebrew folder:
+
+    AR=/usr/local/opt/llvm/bin/llvm-ar CC=/usr/local/opt/llvm/bin/clang make wasm
+    
```

Comment:
I have experienced the error in an Intel Mac and I am unsure about other ones. That's why I wanted to be specific. But I will add that to another commit.

Reply:
yeah but maybe for Intel Macs it could also be in the other location, or the same might be necessary for non-Intel Macs :P unless we know for sure it's best to be generic